apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/SidePanelEditor.utils.tsx

```
@@ -1142,21 +1149,19 @@ export async function insertTableRows({
+          reject(error)
+          return
+        }
 
-          insertProgress = insertProgress + batch.length / rows.length
-          resolve({})
-        }),
-        timeout(30_000),
-      ])
+        insertProgress = insertProgress + batch.length / rows.length
+        resolve({})
+      })
```

Biome (2.4.13)

[error] Promise executor functions should not be `async`.

(lint/suspicious/noAsyncPromiseExecutor)